### PR TITLE
Minor improvement to the getting started code

### PR DESCRIPTION
**Pull request by BenjaminBossan**
- Add missing indentation inside `with` block
- Mark code as Python code for highlighting
- Rename variable name from `dtc_pkl_filename` to `pkl_filename`. The
  name itself doesn't really matter, as it's not assigned anywhere, but
  I found `dtc` confusing here ('decision tree classifier'?).

### CODE REVIEW

**Comment by BenjaminBossan** [comment]
Ready for review @skops-dev/maintainers

**Review by merveenoyan — APPROVED** [review]
LGTM! Can you post before-after example repository for this though?

**Comment by BenjaminBossan** [comment]
@merveenoyan The before repo is just the standard one from the `plot_model_card.py` example. Here it is after the change:

https://huggingface.co/skops/hf_hub_example_getting_started-a88c2f91-5e8e-4d56-9e19-015b40a560d6#how-to-get-started-with-the-model

**Pull request merged**
